utils/prgrib2.F90

```diff
@@ -294,132 +294,132 @@ end subroutine frmt
 !>
 !> @author Stephen Gilbert @date 2010-09-08
 subroutine prvtime(ipdtn, ipdtmpl, listsec1, tabbrev)
-	implicit none
```

Comment:
This reindent desn't look right. We use 4 spaces as the indent, not 7. Also don't use tabs.

Can you adjust your editor for these settings?

Reply:
The warning was for inconsistent indents which was caused by when I started combining some subroutines into one file. I followed the number of spaces from the rest of the files. Which part of the code are you referring to that looks incorrect? The updated code matches the rest of the files as far as I can tell.